test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
@colemickens Good point about the existing Kubernetes e2e tests, I think we should re-orient as soon as possible to re-using existing validation tests and maintain only what is not already covered. That said, I think these changes will improve the e2e tests are in use in acs-engine right now. @jchauncey @dmitsh what do you think?